Classes/DDASLLogCapture.m

```diff
@@ -13,7 +13,7 @@
 //   to endorse or promote products derived from this software without specific
 //   prior written permission of Deusty, LLC.
 
-#import "DDASLLogCapture.h"
+#import <CocoaLumberjack/DDASLLogCapture.h>
```

Comment:
Excerpt from the SE proposal that introduced C-language support:

>When building a target, the package manager will automatically add an additional header search path argument to the include directory for each C language target in the transitive closure of the dependencies for the target being built.
>
>We should use `-iquote` as the header search argument for targets which are within the current package, and `-I` for targets which come from package dependencies. This allows projects to use `#include "foo/foo.h"` versus `#include <foo/foo.h>` syntax appropriately to distinguish between the inclusion of headers which are or are not within the package.

So I think there should be a way to continue including our headers using normal quotes instead of angle brackets.

Reply:
Quote includes in .m files should be fine I think. On the other hand, if they all use the same style it shouldn't matter.

Reply:
@diederich @ffried 
"Include of non-modular header inside framework ..." everywhere.
Yes, `#import "CocoaLumberjack/DDLog.h"` works fine with Swift Package, but it breaks Xcode build.